### PR TITLE
update version of core/identity

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_version.py [hunk 1]
```diff
@@ -3,4 +3,4 @@
 # Licensed under the MIT License.
 # ------------------------------------
 
-VERSION = "1.0.0b5"
+VERSION = "1.0.0"
```
Comment (lmazuel) [inline sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_version.py]: You PR says core and identity, for PR like this, I really want this to be just core and identity

File: sdk/cosmos/azure-cosmos/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
-azure-core<2.0.0,>=1.0.0b4
+azure-core<2.0.0,>=1.0.0
```
Comment (lmazuel) [inline sdk/cosmos/azure-cosmos/requirements.txt]: Why this file exists in the first place? Can we try to delete it?

File: sdk/identity/azure-identity/setup.py [hunk 3]
```diff
@@ -71,7 +71,7 @@
         ]
     ),
     install_requires=[
-        "azure-core<2.0.0,>=1.0.0b5",
+        "azure-core<2.0.0,>=1.0.0",
```
Comment (lmazuel) [inline sdk/identity/azure-identity/setup.py]: Also change Development Status to Stable

Comment (lmazuel) [reply sdk/identity/azure-identity/setup.py]: Exactly:
Development Status :: 5 - Production/Stable

File: sdk/cosmos/azure-cosmos/requirements.txt (hunk 2 above)
Comment (bryevdv) [reply sdk/cosmos/azure-cosmos/requirements.txt]: Seems reasonable, everything in here is already in `shared_requirements.txt`

**Comment by weshaggard** [comment]
FYI @danieljurek @scbedd here are the stable package versions updates.

@xiangyan99 thanks for putting together this PR we were planning to do it from the engineering system team but we can take your change instead. We are working on tooling to update versions so this will give us an idea of the files that need to change as part of that tooling.

File: sdk/cosmos/azure-cosmos/dev_requirements.txt [hunk 4]
```diff
@@ -1,2 +1,2 @@
 -e ../../../tools/azure-sdk-tools
--e ../../core/azure-core
+../../core/azure-core
```
Comment (weshaggard) [inline sdk/cosmos/azure-cosmos/dev_requirements.txt]: Can we remove this from the dev requirements completely now?

Comment (lmazuel) [reply sdk/cosmos/azure-cosmos/dev_requirements.txt]: That should not be part of a version PR, I guess it's fixing something else that was broken at that time.

File: sdk/appconfiguration/azure-appconfiguration/setup.py [hunk 5]
```diff
@@ -87,7 +87,7 @@
     packages=find_packages(exclude=exclude_packages),
     install_requires=[
         "msrest>=0.6.10",
-        "azure-core<2.0.0,>=1.0.0b5",
+        "azure-core<2.0.0,>=1.0.0",
```
Comment (danieljurek) [inline sdk/appconfiguration/azure-appconfiguration/setup.py]: Will setting the core packages to 1.0.0 and updated the dependencies in packages that depend on core cause problems in the nightly runs when azure-core@1.0.0 isn't published?

Comment (xiangyan99) [reply sdk/appconfiguration/azure-appconfiguration/setup.py]: I guess nightly run installs core using -e instead of from pypi?

Comment (lmazuel) [reply sdk/appconfiguration/azure-appconfiguration/setup.py]: It depends how the nightly runs, it should build the wheels from the repo and use that as input. We discussed that with @scbedd already. :)

File: sdk/identity/azure-identity/setup.py [hunk 6]
```diff
@@ -51,7 +51,7 @@
     author_email="azpysdkhelp@microsoft.com",
     url="https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/identity/azure-identity",
     classifiers=[
-        "Development Status :: 4 - Beta",
+        "Development Status :: 5 - Production/Stable",
```
Comment (lmazuel) [inline sdk/identity/azure-identity/setup.py]: @chlowell Also update your HISTORY.txt with the correct date and version

Comment (chlowell) [reply sdk/identity/azure-identity/setup.py]: Done in #8198

**Pull request merged**